docs/UserGuide.md

```diff
@@ -64,6 +64,9 @@ Sellsavvy is a **desktop app for managing contacts, optimized for use via a Comm
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
+
+* Command aliases are alternative words you can use to execute the same command for convenience.<br>
+  e.g. `listcustomer` has the same function as `listc`
```

Comment:
Coudn't comment on line 63 cause its not edited, but `list` is still unchanged.

Reply:
changed